vendor/OWNERS

```diff
@@ -1,10 +0,0 @@
-approvers:
```

Comment:
is this change intentional? if not is there any way to persist this file?

Reply:
Running `dep ensure` will delete that file, since all contents under vendor are assumed to be generated. I'm not sure how to keep that file in. Also I'm not sure if there's a benefit of having an OWNERS file in the vendor directory is, since changes under this directory should be a side effect of code, rather than a code change itself. What do you think?